src/mlpack/methods/preprocess/CMakeLists.txt

```diff
@@ -41,3 +41,9 @@ add_markdown_docs(preprocess_imputer "cli" "preprocessing")
 add_cli_executable(preprocess_scale)
 add_python_binding(preprocess_scale)
 add_markdown_docs(preprocess_scale "cli;python" "preprocessing")
+
+if (STB_AVAILABLE)
+  add_cli_executable(image_converter)
+  add_python_binding(image_converter)
```

Comment:
Should we add the Julia binding too?  (Also, should we do that for `preprocess_scale`?  Looks like maybe I missed that.)

Reply:
Julia binding doesn't seem to work, Should I avoid it

Reply:
I did some digging to debug it.  It's actually an include ordering issue.  If you change the ordering of includes in `preprocess_scale_main.cpp` to this, then it will build right:

```
#include <mlpack/prereqs.hpp>
#include <mlpack/core/util/cli.hpp>
#include <mlpack/core/util/mlpack_main.hpp>
#include <mlpack/core/math/random.hpp>
#include <mlpack/core/math/ccov.hpp>
#include <mlpack/core/data/scaler_methods/max_abs_scaler.hpp>
#include <mlpack/core/data/scaler_methods/mean_normalization.hpp>
#include <mlpack/core/data/scaler_methods/min_max_scaler.hpp>
#include <mlpack/core/data/scaler_methods/pca_whitening.hpp>
#include <mlpack/core/data/scaler_methods/zca_whitening.hpp>
#include <mlpack/core/data/scaler_methods/standard_scaler.hpp>
#include "mlpack/methods/preprocess/scaling_model.hpp"
```

(For mlpack bindings, it has to be `prereqs.hpp` then `cli.hpp` then `mlpack_main.hpp`.  Other orderings cause issues with some binding types.)

Reply:
The same ordering issue *might* be necessary for the image converter; I'm not sure. :+1:

Reply:
Wohhh, How did you arrive to that conclusion, I have made a commit Let's see what happens 

I will go ahead and build it in local tooo :)